app/tests/tests_10/test_generic_api.py

```diff
@@ -100,10 +100,7 @@ def test_invalid_limit_query(self):
                                  msg='Unexpected error message')
 
                 response = self.client.get(f"/{STAC_BASE_V}/{endpoint}?limit=1000")
-                self.assertStatusCode(400, response)
-                self.assertEqual(['limit query parameter too big, must be in range 1..100'],
-                                 response.json()['description'],
-                                 msg='Unexpected error message')
+                self.assertStatusCode(200, response)
```

Comment:
I would add a comment here that the limit is corrected to the max, and maybe check that the limit is corrected also to the configured max.